### PR TITLE
Adds amATOM to Neutron and Osmosis testnets

### PR DESCRIPTION
**Pull request by sinplea**
This PR adds the amATOM token-factory asset to the assetlists of neutrontestnet and osmosistestnet.

### CODE REVIEW

File: testnets/osmosistestnet/assetlist.json [hunk 1]
```diff
@@ -807,6 +807,48 @@
...
+      "name": "amATOM (Testnet)",
+      "display": "amATOM",
+      "symbol": "amATOM",
+      "type_asset": "ics20",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/neutrontestnet/images/amATOM.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/neutrontestnet/images/amATOM.svg"
+      },
+      "images": [
+        {
```
Comment (JeremyParish69) [inline testnets/osmosistestnet/assetlist.json:832]: Please add image sync
```
        "image_sync": {
          "chain_name": "neutrontestnet",
          "base_denom": "factory/neutron15lku24mqhvy4v4gryrqs4662n9v9q4ux9tayn89cmdzldjcgawushxvm76/amatom"
        }
```

Comment (sinplea) [reply testnets/osmosistestnet/assetlist.json:832]: Added :+1:

File: testnets/osmosistestnet/assetlist.json [hunk 2]
```diff
@@ -807,6 +807,52 @@
...
     },
+    {
+      "description": "The yield token for ATOM denominated vaults on Amulet Protocol",
+      "denom_units": [
+        {
+          "denom": "ibc/E972F89F57AF6BDE0CCC97CABFF4DD8D99408B8A741570E1E3D1C35465C98E5E",
+          "exponent": 0
+        },
+        {
+          "denom": "factory/neutron15lku24mqhvy4v4gryrqs4662n9v9q4ux9tayn89cmdzldjcgawushxvm76/amatom",
```
Comment (JeremyParish69) [inline testnets/osmosistestnet/assetlist.json:818]: Please replace this denom with "amATOM"

File: testnets/neutrontestnet/assetlist.json [hunk 3]
```diff
@@ -28,6 +28,33 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/neutrontestnet/images/ntrn.svg"
         }
       ]
+    },
+    {
+      "description": "The yield token for ATOM denominated vaults on Amulet Protocol",
+      "denom_units": [
+        {
+          "denom": "amATOM",
```
Comment (JeremyParish69) [inline testnets/neutrontestnet/assetlist.json:36]: Please swap the denoms (amATOM should be exponent 6, and factory/... should be exponent 0

File: testnets/osmosistestnet/assetlist.json (hunk 2 above)
Comment (sinplea) [reply testnets/osmosistestnet/assetlist.json]: Fixed

File: testnets/neutrontestnet/assetlist.json (hunk 3 above)
Comment (sinplea) [reply testnets/neutrontestnet/assetlist.json]: Fixed :+1:

**Pull request merged**